src/status_im2/contexts/chat/bottom_sheet_composer/gesture.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.gesture
+  (:require
+    [react-native.gesture :as gesture]
+    [react-native.reanimated :as reanimated]
+    [oops.core :as oops]
+    [status-im2.contexts.chat.bottom-sheet-composer.constants :as c]
+    [status-im2.contexts.chat.bottom-sheet-composer.utils :as utils]
+    [utils.re-frame :as rf]))
+
+(defn set-opacity
+  [e opacity translation expanding? min-height max-height new-height saved-height]
```

Comment:
maybe rename `e` to `event`?

Reply:
I am using `e` to be consistent with other calls. I generally prefer to use `e` over `event` for a little more preciseness since we use `oops/oget` which is kinda verbose